src/main/resources/data/computercraft/lua/rom/programs/edit.lua

```diff
@@ -390,8 +418,18 @@ local tMenuFuncs = {
         bRunning = false
     end,
     Run = function()
-        local sTempPath = "/.temp"
-        local ok = save(sTempPath)
+        local sTitle = fs.getName(sPath)
+        if sTitle:sub(-4) == ".lua" then
+            sTitle = sTitle:sub(1, -5)
+        end
+        local sTempPath = bReadOnly and sTitle .. ".temp" or fs.combine(fs.getDir(sPath), sTitle .. ".temp")
+        if fs.exists(sTempPath) then
```

Comment:
Can we call this `".temp." .. sTitle` instead? Just so the file is hidden.

Alternatively go full `(".temp.%s-%08x"):format(sTitle, math.random(0x7FFFFFFF))`, then then don't worry about overwriting files.

Reply:
I don't this there is actually point it going that deep in this case. First filename you suggested will be way more rare then what i used ;p